docs/source/environment.yml

```diff
@@ -13,5 +13,7 @@ dependencies:
   - sphinx_rtd_theme
   - typeguard=2.9.1
   - pip:
+      - hail>=0.2.58
```

Comment:
The bioconda distribution does not seem well maintained. They don't have a version for python 3.7.